src/main/java/Deadline.java

```diff
@@ -0,0 +1,11 @@
+public class Deadline extends Todos {
+
+    protected String type = "D";
+    Deadline(String name, boolean marked, String type) {
```

Comment:
Instead of naming this boolean as 'marked', try to use a more descriptive name to sound more like a boolean. For example, you could rename it as 'isMarked'.